model-engine/model_engine_server/core/aws/storage_client.py

```diff
@@ -1,5 +1,5 @@
 import time
```

Comment:
Can you run the pre-commit hooks? Looks like the unit tests are currently failing on black checks. Otherwise looks good!